package.json

```diff
@@ -110,6 +110,8 @@
     "webpack-node-externals": "^1.5.4"
   },
   "dependencies": {
-    "@hackoregon/civic-server": "0.0.10"
+    "@hackoregon/civic-server": "0.0.10",
+    "victory": "^0.26.0",
+    "webpack-blocks": "^1.0.0"
```

Comment:
Are these only used in the components-library package? If so can these get moved down to that local package.json?

Reply:
Good call. Pushed up a fix, thanks!